src/encoded/static/components/login.js

```diff
@@ -27,33 +25,26 @@ var Login = React.createClass({
         } else { //if logged in give them a logout link
             userActionRender = <a href="#" onClick={this.handleToggle}  data-trigger="logout" className="global-entry">Sign out</a>;
         }
-        return (<div>{userActionRender}</div>);
+        return (
```

Comment:
perhaps add an issue to address this later, but we should probably changing the please wait to use the spin wait icon that is used throughout the rest of the site.